frontend/test/metabase/scenarios/filters/sql-field-filter-id.cy.spec.js

```diff
@@ -0,0 +1,56 @@
+import {
+  restore,
+  mockSessionProperty,
+  openNativeEditor,
+} from "__support__/e2e/cypress";
+
+import * as SQLFilter from "./helpers/e2e-sql-filter-helpers";
+import * as FieldFilter from "./helpers/e2e-field-filter-helpers";
+
+describe("scenarios > filters > sql filters > field filter > ID", () => {
+  beforeEach(() => {
+    restore();
+    cy.intercept("POST", "api/dataset").as("dataset");
+
+    cy.signInAsAdmin();
+    // Make sure feature flag is on regardles of the environment where this is running.
```

Comment:
Spelling then another nit.

```suggestion
    // Make sure feature flag is on regardless of the environment where this is running
```